crates/cubecl-core/src/ir/kernel.rs

```diff
@@ -170,19 +199,35 @@ impl Display for Elem {
             Self::Float(kind) => match kind {
                 FloatKind::F16 => f.write_str("f16"),
                 FloatKind::BF16 => f.write_str("bf16"),
+                FloatKind::Relaxed => f.write_str("minf16"),
```

Comment:
Why not `flex32` since it's a 32 bits type that can be relaxed.

Reply:
I missed that when I renamed it